Umweltzone/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,185 @@
+    <string name="zone_not_drawable_no_email_app">Pas d&#8217;application d&#8217;email compatible d&#8217;installé.</string>
+    <string name="zone_not_drawable_app_chooser_title">Ouvrir le brouillon avec &#8230;</string>
+    <string name="zone_not_drawable_email_subject">"Application Android Umweltzone / Intégration <xliff:g example="Wiesbaden" id="zoneName">%1$s</xliff:g>   "</string>
+    <string name="zone_not_drawable_email_html_message">
+        "<![CDATA[<p>Madame, Monsieur,<br /> <br />
+        Je suis un utilisateur de l&#8217;application libre et open-souce &quot;Umweltzone&quot; [1]
+        qui me permet de rester informé des emplacements des zones de faibles émissions et des régulations
+        propres à chaque ville. Je considère que cette application offre un complément
+        indispensable à l&#8217;offre d&#8217;information des villes.<br /> <br />
+        Malheureusement, les <b>zones de faibles émissions de %1$s</b> ne peuvent pas être affichées.
+        Cela est du au fait que les coordonnées satellites qui en décrivent l&#8217;emplacement n&#8217;ont pas
+        été publiées. Je vous demande donc personnellement de publier ces coordonnées <b>sous licence libre</b>.
+        Pour aider au traitement des données, les <b>formats suivants sont préférables</b> : ESRI Shapefiles,
+        CVS, GeoJSON.<br /> <br />
+        En vous remerciant de votre attention. <br /><br />
+        Cordialement </p>
+        <p> ----<br /><br />
+        Sehr geehrte Damen und Herren,<br /><br />         ich nutze die kostenfreie und quelloffene Anwendung &quot;Umweltzone&quot; [1],         mit der ich mich über den Verlauf von Umweltzonen und individuelle         Regelungen in verschiedenen Städten auf meinem Android-Gerät         informieren kann. Ich betrachte diese Anwendung als nützliche         Erweiterung zum bestehenden Informationsangebot vieler Städte.<br />         <br />         Leider kann die <b>Umweltzone %1$s</b> nicht dargestellt werden. Es fehlen         die Geokoordinaten des Umweltzonenumrings. Daher möchte ich Sie hiermit         persönlich bitten, die Koordinaten unter einer <b>freien Lizenz</b> zu         veröffentlichen. Um eine Weiterverarbeitung zu ermöglichen, eignen         sich folgenden <b>maschinenlesbare Dateiformate</b>:         ESRI Shapefiles, CSV, GeoJSON.<br />         <br />         Mit freundlichen Grüßen</p>          <p>___<br />         [1]: <a href="https://play.google.com/store/apps/details?id=de.avpptr.umweltzone">https://play.google.com/store/apps/details?id=de.avpptr.umweltzone</a></p>]]>     "</string>
+    <string name="trace_droid_dialog_title">"Envoyer un rapport de plantage &#8230;   "</string>
+    <string name="trace_droid_dialog_message">
+        "Un problème avec l&#8217;application Umweltzone a été détecté. Il serait très utile de nous envoyer
+        un rapport de plantage pour que nous puissions identifier la cause du problème."</string>
```

Comment:
Please remove the wrapping double quotes (`"`) and the unneeded whitespace. These cause formatting issues.